src/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -38,13 +38,51 @@ trade-offs of pre-warming an engine.
 ### Create a FlutterEngine
 
 The proper place to create a `FlutterEngine` is specific
-to your host app. As an example, we demonstrate creating a
-`FlutterEngine`, exposed as a property, on app startup in
-the app delegate.
+to your host app. 
 
 {% samplecode engine %}
 
-{% sample Swift %}
+{% sample SwiftUI %}
+As an example, we demonstrate creating a
+`FlutterEngine` inside an `ObservableObject`, 
+and passing that into a `View` using the 
+ `environmentObject()` modifier. 
```

Comment:
**issue:** This section should indicate what's done where. This throws in a lot of information with little context.
```suggestion
In this example, we create a `FlutterEngine` object inside a SwiftUI `ObservableObject`. 
We then pass this `FlutterEngine` into a `ContentView` using the 
 `environmentObject()` property. 
```

**suggestion:** It would not hurt to link out to the [SwiftUI docs](https://developer.apple.com/documentation/swiftui/environmentobject) in this case to make sure the dev can follow along.